docs/components/modeler/desktop-modeler/element-templates/defining-templates.md

```diff
@@ -661,3 +666,75 @@ Per default, the element template defines the visible entries of the properties
 ```
 
 ![Display default entries](./img/entries-visible.png)
+
+### Defining Conditional Properties
+
+Properties may have a condition which determines when they should be active, depending on the value of another property.
```

Comment:
Quick feedback.

IMO we should explain what `active` means. E.g.;

If a property is `active` it is displayed in the properties panel.
If a property is not `active` it is not displayed in the properties panel. Also the value of that property is de-serialzied from the XML.

Reply:
Makes sense, I will add a sentence for this